app/src/main/res/values-en/strings.xml

```diff
@@ -1 +0,0 @@
-../values/strings.xml
```

Comment:
i do not get why this file was changed. can you please explain

Reply:
I deleted it arbitrarily because it was a string.xml file with nothing. Sorry that made it impossible to run on my computer.

Reply:
This is a symlink. values-en/strings.xml points to ../values/strings.xml because english translation is our default language.

Reply:
I put it back.

Reply:
Here the TRAVIS CI shows all checks have failed. Can I get a review without this file for a while?